src/session/session_compact.c

```diff
@@ -209,6 +213,7 @@ __compact_checkpoint(WT_SESSION_IMPL *session)
     /* Checkpoints take a lot of time, check if we've run out. */
     WT_RET(__wt_session_compact_check_timeout(session));
 
+    WT_STAT_CONN_INCR(session, session_table_compact_checkpoint);
```

Comment:
Would it be more useful to see when a checkpoint is running? Maybe we can increment it here and decrement after `__wt_txn_checkpoint`, same as you did `session_table_compact_running`?